docker/build_fake_dbs.sh

```diff
@@ -9,10 +9,10 @@ conda activate icubam
 pwd
 ls -a
 echo "Build test db"
-PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config.toml --mode=dev
+PYTHONPATH=. python scripts/populate_db_fake.py --config=resources/config_dev.toml
```

Comment:
To go along with @pgraverdy's comment, I feel like we should just have a single config file.  The prod/dev/test whatever approach is not something I've seen in many other software packages, and can be easily dealt with by simply allowing for /whatever/ config file you would want.